kubernetes/single-tenant/README.md

```diff
@@ -0,0 +1,4 @@
+## Script defaults
```

Comment:
Please provide more details here. You can just copy what you wrote on the main kubernetes/README.md.